tests/system/apps/fake-framework.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "/fake-framework",
-  "cmd": "/opt/mesosphere/bin/python -m http.server $PORT0",
+  "cmd": "sleep 5 && /opt/mesosphere/bin/python -m http.server $PORT0",
```

Comment:
I don't get how the various times relates here. Why do we need the sleep here?

Reply:
hmm maybe I get it now ... I think it would deserve a comment somewhere, what do you think?

Reply:
The readiness check might start immediately and the app would become ready.